vcd/resource_vcd_external_network_v2.go

```diff
@@ -47,23 +47,17 @@ var networkV2IpScope = &schema.Resource{
 			Optional:     true,
 			Description:  "Primary DNS server",
 			ValidateFunc: validation.IsIPAddress,
-			// Only NSX-V allows configuring DNS
-			ConflictsWith: []string{"nsxt_network"},
 		},
 		"dns2": {
 			Type:         schema.TypeString,
 			Optional:     true,
 			Description:  "Secondary DNS server",
 			ValidateFunc: validation.IsIPAddress,
-			// Only NSX-V allows configuring DNS
-			ConflictsWith: []string{"nsxt_network"},
 		},
 		"dns_suffix": {
 			Type:        schema.TypeString,
 			Optional:    true,
 			Description: "DNS suffix",
-			// Only NSX-V allows configuring DNS
-			ConflictsWith: []string{"nsxt_network"},
```

Comment:
As we are now lifting the validation you had previously for a reason, do we open up a possibility for misuse? If yes, does the API throw an error in that case and is the error descriptive enough?

Reply:
Yes. It was stated, that this was supposed to be only for NSX-V networks, however, it is not the case. Note, that as I stated in PR description - `dns_suffix` field still has a bug in VCD and it is not being set, but this is to be fixed in future VCD versions.